obspy/signal/trigger.py

```diff
@@ -428,6 +428,46 @@ def pk_baer(reltrc, samp_int, tdownmax, tupevent, thr1, thr2, preset_len,
         return pptime.value + 1, pfm.value.decode('utf-8')
 
 
+def aic(td):
+    """
+    Differently from AR-AIC picker, this function calculates AIC function
+    directly from the data, without using the AR coefficients (Maeda, 1985)
+
+    Computes P-phase arrival time digital single-component acceleration
+    or broadband velocity record without requiring threshold settings using
+    AKAIKE INFORMATION CRITERION.
+    Returns P-phase arrival time index and the characteristic function.
+    The returned index correspond to the characteristic function's minima.
+
+    :type td: numpy.ndarray
+    :param td: time series as numpy.ndarray float32 data
```

Comment:
It should not matter what dtype the data comes in, does it?

Reply:
You are right, it was because of old assertion I had in my project. Now the function checks if the input time-series is an `np.ndarray` class. No checks on dtypes anymore. Docstring updated.